cli_client/python/timesketch_cli_client/commands/timelines.py

```diff
@@ -36,19 +36,50 @@ def list_timelines(ctx):
 
 @timelines_group.command("describe")
 @click.argument("timeline-id", type=int, required=False)
+@click.option(
+    "--output-format",
+    "output",
+    required=False,
+    help="Set output format (overrides global setting).",
```

Comment:
Maybe describe in the help which output formats are available. In this case only json or text I assume?